stablehlo/reference/Ops.cpp

```diff
@@ -289,7 +268,7 @@ Tensor evalReverseOp(const Tensor &operand, ArrayRef<int64_t> dimensions,
   auto resultShape = result.getType().getShape();
   for (auto resultIt = result.index_begin(); resultIt != result.index_end();
        ++resultIt) {
-    SmallVector<int64_t> operandIdx(*resultIt);
+    Index operandIdx(*resultIt);
     for (auto dim : dimensions)
       operandIdx[dim] = (resultShape[dim] - 1) - operandIdx[dim];
```

Comment:
done.

Reply:
Actually I was wrong - we cannot use helper methods here because this logic is only meant to affect a subset of axes. Our tests didn't catch this because we only have one test which reverses all axes :)

Reply:
That is right. I missed that.
Reverted that change.